django-drf-react-quickstart/mcbcapp/frontend/src/components/IntroPage.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class IntroPage extends Component{ 
+    render(){                   
+        const backgroundStyle = {
+            backgroundColor: "#32669D",
+            width: "100vw",
+            height: "100vh",
+        };
+        const divStyle = {
+            height: "35%",
+            width: "60%",
+            position: "absolute",
+            top: "25%",
+            left: "20%"
+
+        }
+        const nameStyle1 = {
+            color: "white",
+            fontSize: "7.5vw", 
+            textAlign: "center"              
+        };
+        const nameStyle2 = {
+            color: "white", 
+            fontSize: "3vw",
+            textAlign: "center"              
+        };
+        const imgStyle = {
+            display: "block",
+            margin: "auto"
+        }
+        return(
+            <body style={backgroundStyle}>
```

Comment:
your whole component will already be nested in a **body** tag so this actually isnt needed

Reply:
use div instead